demos/showcase/showcase-webapp/src/main/webapp/views/component.xhtml

```diff
@@ -41,6 +41,23 @@
 					useWrapMode="false" height="800" width="100%" value="#{useCaseCodeExample.rawText}" />
 			</alloy:tab>
 		</alloy:tabView>
+		<alloy:buttonRow class="text-center">
```

Comment:
@stiemannkj1 FYI, after downgrading to JSF 2.1 Mojarra started complaining about this line. I'll be changing the class attribute to styleClass shortly.